components/Faq.vue

```diff
@@ -69,33 +36,97 @@ export default {
       required: true
     }
   },
-  computed: {
-    sortedItems() {
-      return [...this.items].sort((a, b) => {
-        if (a.question === 'What is a hackathon?') {
-          return -1
-        } else if (b.question === 'What is a hackathon?') {
-          return 1
-        } else {
-          return 0
-        }
-      })
+  data() {
+    return {
+      leftFaqs: this.getFaqColumn(true),
+      rightFaqs: this.getFaqColumn(false)
     }
   },
   methods: {
-    openSesame: (e) => {
+    toggleBox: (e) => {
       e.target.classList.toggle('active')
       const panel = e.target.nextElementSibling
       if (panel.style.display === 'block') {
         panel.style.display = 'none'
       } else {
         panel.style.display = 'block'
       }
+    },
+    getFaqColumn(left) {
+      const isLeft = (e, i) => left ? i % 2 === 0 : i % 2 !== 0
+      return this.items.filter(isLeft)
+    },
+    // For future usage
+    filterFaq: (category) => {
+      return this.items.filter(i => i.category === category)
+    },
+    prioritizeQuestion: () => {
```

Comment:
Should this function be called somewhere because right now it's not.

Reply:
It's for future use since we want 'What is a hackathon' to be the first faq, maybe this will be handy should design want 'What is build day?'/ 'what is learn day?' eventually.